server/matchmaker/search.py

```diff
@@ -308,3 +336,9 @@ def cancel(self):
 
     def __str__(self):
         return "CombinedSearch({})".format(",".join(str(s) for s in self.searches))
+
+    def get_original_searches(self) -> List[Search]:
+        """
+        Returns the searches of which this CombinedSearch is comprised
+        """
+        return list(self.searches)
```

Comment:
What if you have something nested like `CombinedSearch([CombinedSearch([...], Search())`? Should it flatten the searches out like this? Or just go down one layer?
```suggestion
        return list(s for search in self.searches for s in search.get_original_searches())
```

Reply:
I only want to go down one layer. I don't know a better wording for the doc string, though.